app/controllers/houston/dashboards/staging_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Houston::Dashboards::StagingController < ApplicationController
   layout "houston/dashboards/dashboard"
-  STAGING_PROJECTS = %w(members unite ledger bsb)
+  STAGING_PROJECTS = %w(members unite ledger bsb members-dav)
```

Comment:
I've put in a fix for this using `github.org_issues`. Houston doesn't currently pull in the `body` of the PR and that's where I pull the checkbox data from. My most recent commit is still a nice improvement over my previous solution.

If you still prefer querying `Github::PullRequest` instead of GitHub, I can create a migration to add the body to the model _or_ I could use the same logic to find how many check boxes are checked vs. unchecked and then add those two columns to `Github::PullRequest` instead of the whole body.